spec.go

```diff
@@ -36,6 +36,10 @@ type TorrentSpec struct {
 	// Whether to allow data download or upload
 	DisallowDataUpload   bool
 	DisallowDataDownload bool
+
+	// Custom encoder for webseed URLs
+	// Leave nil to use the default (url.QueryEscape)
+	EncodeWebSeedUrl func(string) string
```

Comment:
How about `DefaultWebseedEscapePath`, where default here means the escaping that is used when webseed URLs are given through API constructs that don't take an option to specify an alternative escaping?

Reply:
Done https://github.com/afjoseph/torrent/commit/a79c3bd5d30d99fb33ab7c5334c7d017884d4403

Reply:
I think it's actually better to avoid the extra option just for webseeds entirely (since each URL can require different options).